tests/pxScene2d/testRunner/tests.json

```diff
@@ -4,7 +4,8 @@
       {"url": "http://pxscene.org/examples/px-reference/gallery/gallery3.js", "title": "gallery3", "timeToRun": "5000", "useBaseURI": "false" },
       {"url":"../tests/simpleTestApi.js", "title":"simpleTestApi","useBaseURI":"true"},
       {"url":"../tests/textbox_auto_v3.js", "title":"textbox auto","useBaseURI":"true"},
-      {"url":"../tests/text2tests_v8.js", "title":"text2tests","useBaseURI":"true"},
+      {"url":"../tests/text2tests_v9.js", "title":"text2tests","useBaseURI":"true"},
+      {"url":"../tests/text2tests_more_v2.js", "title":"text2tests","useBaseURI":"true"},
```

Comment:
Please change the title value for this test to be "text2tests_more" so as not to be confused with the "text2tests" tests.

Reply:
Done. Thanks